buildspec.yaml

```diff
@@ -12,7 +12,7 @@ phases:
       - /bin/bash ./build.sh
   post_build:
     commands:
-      - "[ -f .BUILD_COMPLETED ] && /bin/bash ./pack.sh"
+      - "[ -f .BUILD_COMPLETED ] && [ "${SKIP_PACK:-}" != "YES PLEASE" ] && /bin/bash ./pack.sh"
```

Comment:
Why not keep it simple? `[ -z "${SKIP_PACK}" ]`

Reply:
Because I want to be absolutely certain that you totally intend on skipping `pack.sh`. I'm fine with the `-z` if you insist :)

Reply:
Sounds like pool proofing of the wrong kind, but I don't really care.